hologram_dashboard/fetcher/hologram_data_source_availability_fetcher_test.cc

```diff
@@ -0,0 +1,48 @@
+/*
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*/
+
+#include "hologram_data_source_availability_fetcher.h"
+#include <google/protobuf/util/message_differencer.h>
+
+namespace wireless_android_play_analytics{
+
+TEST(FetcherTest, AcquireConfigTest) {
+    // Test malformed/invalid path.
+    HologramDataSourceAvailabilityFetcher hologram_fetcher;
+    ASSERT_DEATH(hologram_fetcher.AcquireConfig("/abc"), "");
+    ASSERT_DEATH(hologram_fetcher.AcquireConfig(
+        "fetcher/testdata/hologram_config_malformed.config"), "");
+
```

Comment:
Split this test into two test cases. 
One for malformed input path, the other for success case.